src/main/java/com/github/dockerjava/api/model/Frame.java

```diff
@@ -28,6 +28,7 @@ public String toString() {
         return String.format("%s: %s", streamType, new String(payload).trim());
     }
 
+    //CHECKSTYLE:OFF
```

Comment:
autogenerated equals and hashcode doesn't match rules, so i just disabled checks